test/WebSites/ActionResultsWebSite/Controllers/ActionResultsVerificationController.cs

```diff
@@ -108,5 +129,46 @@ private DummyClass CreateDummy()
                 SampleString = "Foo"
             };
         }
+
+        private DisposableDummy CreateDisposableDummy(string guid)
+        {
+            return new DisposableDummy(Service, guid);
+        }
+
+        private class DisposableDummy : IDisposable
+        {
+            private GuidLookupService _service;
+            private string _guid;
+
+            public DisposableDummy(GuidLookupService service, string guid)
+            {
+                _service = service;
+                _guid = guid;
+                _service.IsDisposed[_guid] = false;
+            }
+
+            public void Dispose()
+            {
+                _service.IsDisposed[_guid] = true;
+            }
+        }
+    }
+
+    public class GuidLookupService
+    {
+        private Dictionary<string, bool> _lookup;
+
+        public GuidLookupService()
+        {
+            _lookup = new Dictionary<string, bool>();
+        }
+
+        public Dictionary<string, bool> IsDisposed
+        {
+            get
+            {
+                return _lookup;
+            }
+        }
```

Comment:
This could just be `public Dictionary<string, bool> IsDisposed { get; } = new Dictionary<....`